monkey/common/agent_configuration/agent_configuration.py

```diff
@@ -15,7 +15,7 @@ class AgentConfiguration(MutableInfectionMonkeyBaseModel):
         "seconds)",
         default=30,
     )
-    credentials_collectors: Tuple[PluginConfiguration, ...] = Field(
+    credentials_collectors: Optional[CredentialsCollectorsConfiguration] = Field(
```

Comment:
Make this match `ExploitationConfiguration.exploiters`. It shouldn't be optional and we don't need to define a separate configuration.